engine/extensions/src/test/java/org/pentaho/reporting/engine/classic/extensions/modules/java14print/Java14PrintUtilTest.java

```diff
@@ -45,7 +45,7 @@
 
 import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.*;
```

Comment:
Let's expand that to only the classes we're using.

Reply:
expanded